scripts/seed-database/write/published_flows.sql

```diff
@@ -29,7 +29,7 @@ SELECT
   publisher_id,
   created_at,
   has_send_component
-FROM sync_published_flows
+  FROM sync_published_flows
```

Comment:
nit: lots of trailing spaces and atypical sql indents introduced here which decrease readability - we don't have linting on .sql files so please try to pay closer attention to manual changes here to avoid this !